### PR TITLE
Make the format of doc comments consistent

### PR DESCRIPTION
**Pull request by ManaSugi**
Currently, the format of doc comments is not consistent.
Hence, this commit decides the format as follows in the crate.
This format is based on https://doc.rust-lang.org/rust-by-example/meta/doc.html

The doc comments of function:
```
/// Brief summary of the function on one line. (Start from "verb-s")
///
/// Detailed description of the function. (if you have)
///
/// Description of return value on success.
/// (Start from "This function returns...")
///
/// # Arguments (if the function has arguments)
///
/// * `arg` - Descripition of the argument
///
/// # Errors (if the function returns `Result`)
///
///  If this function encounters..., an error will be returned.
///
/// # Panics (if the function can `panic`)
///
///  Panics if...
///
/// # Examples (if you want to show the example)
///
/// ```
/// Example codes
/// ```
pub fn foo
```

Signed-off-by: Manabu Sugimoto <Manabu.Sugimoto@sony.com>

### CODE REVIEW

**Comment by ManaSugi** [comment]
I added `do-not-merge` label because we should refactor doc comments about `notification`(#45)
according to this format at first. Then, this PR will be merged into main.
@rust-snake Do you have any suggestions about the doc format?

**Comment by rusty-snake** [comment]
> Do you have any suggestions about the doc format? :)

The format looks good.

Other common section we could consider

```
# Safety (for unsafe functions)

# Panics (for function that panic!, assert!, unwrap, ...)

# Errors (for Result/Option)
```

**Comment by ManaSugi** [comment]
@rusty-snake Thank you for your suggestions!

The current format does not have `Errors` section explicitly because the descripition is included in the `Description of return value`. However, as you suggested, it is better to create `Errors` section if the function returns `Result`, so I'll fix it as follows.
What do you think?

The format of doc comments: 
```
/// Brief summary of the function on one line. (Start from "verb-s")
///
/// Detailed description of the function. (if you have)
///
/// Description of return value on success. 
/// (Start from "This function returns...")
///
/// # Arguments (if the function has arguments)
///
/// * `arg` - Descripition of the argument
///
/// # Errors (if the function returns `Result`)
/// 
///  If this function encounters..., an error will be returned.
///
/// # Panics (if the function can `panic`)
///
///  Panics if...
///
/// # Examples (if you want to show the example)
///
/// ```
/// Example codes
/// ```
pub fn foo
```

File: libseccomp/src/lib.rs
Comment (rusty-snake) [inline libseccomp/src/lib.rs:955]:
```suggestion
    /// See the [seccomp_attr_set(3)] man page for details on available attribute values.
    /// [seccomp_attr_set(3)]: https://www.man7.org/linux/man-pages/man3/seccomp_attr_set.3.html
```

File: libseccomp/src/lib.rs
Comment (rusty-snake) [inline libseccomp/src/lib.rs:1011]: export_pfc uses the abrevation (PFC) while export_bpf uses the long form (Berkeley Packet Filter).

I'm not sure which one is better, but this is a bit inconsistent.

File: libseccomp/src/lib.rs
Comment (rusty-snake) [inline libseccomp/src/lib.rs:1145]:
```suggestion
/// This function returns a string containing the name of the syscall.
```

File: libseccomp/src/lib.rs
Comment (rusty-snake) [inline libseccomp/src/lib.rs:92]:
```suggestion
/// Represents filter attributes.
///
/// You can set/get the attributes of a filter context with it's [`set_filter_attr`] and [`get_filter_attr`] methods.
```

File: libseccomp/src/lib.rs
Comment (rusty-snake) [inline libseccomp/src/lib.rs:610]:
```suggestion
/// **Represents a filter context in the libseccomp.**
```

File: libseccomp/src/lib.rs
Comment (ManaSugi) [reply libseccomp/src/lib.rs]: Thank you! I fixed it.

File: libseccomp/src/lib.rs
Comment (ManaSugi) [reply libseccomp/src/lib.rs]: Thank you! I fixed it.

File: libseccomp/src/lib.rs
Comment (ManaSugi) [reply libseccomp/src/lib.rs]: I use `PFC(Pseudo Filter Code)-formatted` and `BPF(Berkeley Packet Filter)-formatted` for the consistent.

**Review by rusty-snake — APPROVED** [review]
LGTM

**Comment by ManaSugi** [comment]
Thank you for the approve. I'll merge this after merging #45.

**Pull request merged**
